pkg/container/types/embedding.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2023 Matrix Origin
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package types
+
+import (
+	"fmt"
+	"github.com/matrixorigin/matrixone/pkg/common/moerr"
+	"math"
+	"strings"
+	"unsafe"
+)
+
+const (
+	// MaxVecDimension Based on here: https://github.com/pgvector/pgvector/blob/b56971febeec389a011de7bb40b3349e24757aff/src/vector.h#L10
+	MaxVecDimension = 16000
```

Comment:
This is so arbitrary.    We do not need to inherit arbitrary limit from pgvector.

I would suggest 32768 or 65536.

Reply:
sure.